ding/model/template/tests/test_hpt.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import torch
+from itertools import product
+from ding.model.template.hpt import HPT
+from ding.torch_utils import is_differentiable
+
+T, B = 3, 4
+obs_shape = [4, (8, ), (4, 64, 64)]
```

Comment:
Remove the group `(4, 64, 64)` here, the current HPT model you implemented here can't support the image input like `(4, 64, 64)`. It need a CNN feature extractor, and the corresponding `state_dim` should be `(4, 64, 64) rather than the current scalar. We will left this part for future, thus you can remove this group now.